### ISSUE TITLE
Does not trigger events

### ISSUE THREAD

**Issue by gunn** [tuvistavie/atomic-chrome#21]
Great project! It would be even nicer if when a textfield was edited, the `onchange` event fired.

My use case is for editing markup that is previewed as html and updates live with typing.

**Comment by danhper** [comment]
@gunn Hi, thanks for the feedback!
For now [the keypress event is fired](https://github.com/tuvistavie/atomic-chrome/blob/master/src/content-script-tools/custom-events/common.js#L8), but I am not sure that firing the `onchange` event everytime
is a great idea, as it differs from the normal behavior of a textarea or input, where it fires only
when the input is unfocused. Maybe we could use a keyboard shortcut to fire it from Atom?
What do you think?

**Comment by gunn** [comment]
@tuvistavie yes I think you're right that always firing `onchange` would be inconsistent.

The reason I was thinking that is because of [React's onChange attribute](https://facebook.github.io/react/docs/forms.html). 

Here's a very simple test-case for working with a React textarea - http://output.jsbin.com/funonayaki Source [here](http://jsbin.com/funonayaki/edit?js,output). You can see it updates correctly when typing in the browser, but not from atom. Maybe the `oninput` event would be the correct one?
